README.md

```
@@ -110,6 +113,15 @@ it is possible to pass functions with args and kwargs using tuple, dict or `Flag
 
 ## Change Log
 
+### 1.0.0
+ * **BREAKING CHANGE**: bump dependencies, this version requires `UnleashClient>=6.4.0`
+
+### 0.4.4
+ * add a variable for fake initialization
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)